cetmix_tower_server/models/cx_tower_server.py

```diff
@@ -1319,7 +1319,7 @@ def _command_runner_flight_plan(
             # Generate custom label and add values for log
             kwargs["plan_log"] = {
                 "label": generate_random_id(4),
-                "parent_flight_plan_log_id": log_record.plan_log_id.id,
```

Comment:
Shouldn't we keep the parent flight plan log too?

Reply:
now, this fields is related field:

```python
related="parent_command_log_id.plan_log_id",
```

so, we can just save the `parent_command_log_id` field there and the parent_flight_plan_log_id will calculated